std/json.d

```diff
@@ -1700,7 +1701,10 @@ pure nothrow @safe @nogc unittest
 
 pure nothrow @safe unittest // issue 15884
```

Comment:
If you want to add support for static arrays, you will need fix the constructor, because otherwise the assignment from rvalue static array in the above test won't compile.